libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/InternalUtils.kt

```diff
@@ -1,27 +1,49 @@
 package com.mapbox.navigation.core.internal.utils
 
-import com.mapbox.navigation.core.trip.session.MapboxTripSession
+object InternalUtils {
 
-/**
- * Internal API used for testing. Sets the static unconditional polling patience value for all
- * instances in the process.
- *
- * Pass `null` to reset to default.
- *
- * Do not use in a production environment.
- */
-fun setUnconditionalPollingPatience(patience: Long?) {
-    MapboxTripSession.UNCONDITIONAL_STATUS_POLLING_PATIENCE = patience ?: 2000L
-}
+    @Volatile
+    internal var LOOK_AHEAD_SECONDS = 1.0
+
+    @Volatile
+    internal var UNCONDITIONAL_POLLING_PATIENCE_SECONDS = 2.0
+
+    @Volatile
+    internal var UNCONDITIONAL_POLLING_INTERVAL_SECONDS = 1.0
+
+    /**
+     * Internal API used for testing. Sets the static look ahead value for all
+     * instances in the process. Needs to be set prior to [MapboxNavigation] creation.
+     *
+     * Pass `null` to reset to default.
+     *
+     * Do not use in a production environment.
+     */
+    fun setLookAhead(lookAheadSeconds: Double?) {
+        LOOK_AHEAD_SECONDS = lookAheadSeconds ?: 1.0
+    }
+
+    /**
+     * Internal API used for testing. Sets the static unconditional polling patience value for all
+     * instances in the process. Needs to be set prior to [MapboxNavigation] creation.
+     *
+     * Pass `null` to reset to default.
+     *
+     * Do not use in a production environment.
+     */
+    fun setUnconditionalPollingPatience(patienceSeconds: Double?) {
+        UNCONDITIONAL_POLLING_PATIENCE_SECONDS = patienceSeconds ?: 2.0
```

Comment:
@Guardiola31337 It looks like previously this method accepted `patience: Long?` milliseconds value, and now it accepts `patienceSeconds: Double?` seconds. So if the consumers of this interface are not notified about this change - they could get broken behavior (e.g., they could just continue to call `setUnconditionalPollingPatience(1000)` - this will be compiled, but this will break the old behavior, because now it means 1000 seconds, not 1000 milliseconds). Suggestions:
1. Don't update this interface, keep old `setUnconditionalPollingPatience(patience: Long?)` signature. Convert the incoming `milliseconds` value into `seconds` inside the method.
2. Add documentation about `seconds` or `milliseconds` value usage - so this is declared explicitly.

Reply:
I'd be behind leaving the milliseconds in the public API and converting the value internally into seconds.

Reply:
@DmitryAk Going to refactor in favor of

> 1. Don't update this interface, keep old `setUnconditionalPollingPatience(patience: Long?)` signature. Convert the incoming `milliseconds` value into `seconds` inside the method.

In any case, why did that change internally? I'd expect that NN also accepts milliseconds so we don't need to do any conversion from the SDK side ¯\\\_(ツ)\_/¯

Reply:
@LukasPaczos @DmitryAk I've just pushed a commit addressing this. Ready for another round of 👀

Reply:
@Guardiola31337 I cannot find: how were these values accepted by NN previously (in milliseconds)? The only previous usage I found is `delay(UNCONDITIONAL_STATUS_POLLING_PATIENCE)`, but it is internal to Android - so it looks like these values were not accepted by NN previously.

Reply:
Yeah but now they're requested from `PollingConfig`. Should these be in milliseconds too? Currently they're in seconds.

Reply:
I'd vote for nanoseconds - it is used for location timestamp, feels more natural to make it nanoseconds too then

Reply:
Happy to change in the next release when that fix is available from NN 👍